README.md

```diff
@@ -97,37 +95,44 @@ PresentationModel instance is automatically retained during configuration change
 Lifecycle callbacks:
 - `onCreate()` — Called when the PresentationModel is created. Initialize your Rx chains in this method.
 - `onBind()` — Called when the View binds to the PresentationModel.
+- `onResume` - Called when the View resumes and begins to receive updates from states and commands.
+- `onPause` - Called when the View pauses. At this point, states and commands stops emitting to the View and turn on internal buffer until the View resumes again. 
 - `onUnbind()` — Called when the View unbinds from the PresentationModel.
 - `onDestroy()` — Called when the PresentationModel is being destroyed. Dispose all subscriptions in this method.
 
 What's more you can observe lifecycle changes via `lifecycleObservable`.
 
-Also the useful extensions of the *Disposable* are available to make lifecycle handling easier: `untilUnbind` and `untilDestroy`. 
+Also the useful extensions of the *Disposable* are available to make lifecycle handling easier: `untilPause`,`untilUnbind` and `untilDestroy`.
 
 ### PmView
-The library has several predefined PmView implementations: `PmSupportActivity`, `PmSupportFragment` and `PmController` (for [Conductor](https://github.com/bluelinelabs/Conductor/)'s users).  
+The library has several predefined PmView implementations: `PmActivity`, `PmFragment`, `PmDialogFragment` and `PmController` (for [Conductor](https://github.com/bluelinelabs/Conductor/)'s users).  
 
 You have to implement only two methods:
 1) `providePresentationModel()` — Create the instance of the PresentationModel.
-2) `onBindPresentationModel()` — Bind to the PresentationModel properties in this method. Use the `bindTo` extension and [RxBinding](https://github.com/JakeWharton/RxBinding) for this. 
-
-Also there is variants of these with Google Map integration.
+2) `onBindPresentationModel()` — Bind to the PresentationModel properties in this method. Use the `bindTo`, `passTo` extensions and [RxBinding](https://github.com/JakeWharton/RxBinding) to do this.
 
 ### State
 **State** is a reactive property which represents a View state.  
 It holds the latest value and emits it on binding. For example, **State** can be used to represent a progress of the http-request or some data that can change in time.
 
 In the PresentationModel:
 ```kotlin
-val inProgress = State<Boolean>(false)
+val inProgress = state(false)
 ```
-Change the value through the consumer:
+Change the value:
 ```kotlin
-inProgress.consumer.accept(true)
+inProgress.accept(true)
 ```
 Observe changes in the View:
 ```kotlin
-pm.inProgress.observable bindTo progressBar.visibility()
+pm.inProgress bindTo progressBar.visibility()
+```
+Usually there is already a data source or the state is derived from other states. In this case, it’s convenient to describe this using lambda like as shown below:
```

Comment:
Usually there is a data source already or the state is derived from other states. In this case, it’s convenient to describe this using lambda as shown below: